### PR TITLE
Cleanup api docs for Tina Cloud and CLI

### PR DESCRIPTION
**Pull request by jeffsee55**
Closes https://github.com/tinacms/tina-cloud-starter/issues/46

### CODE REVIEW

File: content/toc-doc.json [hunk 1]
```diff
@@ -378,15 +378,61 @@
...
+        ]
+      },
+      {
+        "id": "/docs/tina-cloud/client",
+        "slug": "/docs/tina-cloud/client",
+        "title": "Tina Cloud client",
+        "items": [
+          {
+            "id": "/docs/tina-cloud/client/#usegraphqlforms",
+            "href": "/docs/tina-cloud/client/#usegraphqlforms",
```
Comment (jeffsee55) [inline content/toc-doc.json:422]: @dwalkr this is reverting back to the global menu. I can't quite tell which key I'm supposed to use here

**Review by jamespohalloran — APPROVED** [review]

File: content/docs/tina-cloud/cli.md [hunk 2]
```diff
@@ -45,16 +45,19 @@ The simplest way to get started is to add a `.tina/schema.ts` file
 mkdir .tina && touch .tina/schema.ts
 ```
 
-Inside this file, we'll define our schema. This is schema will generate a GraphQL API which works specifically for your content.
+### `defineSchema`
+
+`defineSchema` is tells the CMS how to build your content API.
```
Comment (DirtyF) [inline content/docs/tina-cloud/cli.md:50]:
```suggestion
`defineSchema` tells the CMS how to build your content API.
```

File: content/docs/tina-cloud/client.md [hunk 3]
```diff
@@ -182,10 +184,44 @@ const [payload, isLoading] = useGraphqlForms({
...
+import { useDocumentCreatorPlugin } from 'tina-graphql-gateway'
+
+// args is of type:
+// collection: {
+//   slug: string;
+// };
+// relativePath: string;
+// breadcrumbs: string[];
+// path: string;
+useDocumentCreatorPlugin(args => window.location.assings(buildMyRouter(args)))
```
Comment (DirtyF) [inline content/docs/tina-cloud/client.md:201]:
```suggestion
useDocumentCreatorPlugin(args => window.location.aiign(buildMyRouter(args)))
```

**Comment by logan-anderson** [comment]
Noticing some weird nav behaviour (https://www.loom.com/share/1cc32aae57004522a46cc2b0128f63e1)

File: content/toc-doc.json (hunk 1 above)
Comment (dwalkr) [reply content/toc-doc.json:422]: @jeffsee55 url matching wasn't playing nice with hash URLs. pushed up a fix to this branch.

**Review by dwalkr — APPROVED** [review]

**Pull request merged**